utils/getFormatsDescription.ts

```diff
@@ -0,0 +1,64 @@
+import { Formats } from "@/schemas/collections.schemas";
+import { startCase } from "lodash";
+
+export default function getFormatsDescription(formats: Formats) {
+  if (formats.length === 1) {
+    return formats
+      .map((format) => {
+        const { name, text, descriptions } = format;
+        const qty = parseInt(format.qty);
+
+        if (text && descriptions) {
+          return qty > 1
+            ? startCase(`${text} ${name} ${descriptions?.join(" ")})`) +
```

Comment:
you can nest template literals, I think it's more readable/concise than concatination: 

```ts
startCase(`${text} ${name} ${descriptions?.join(' ')})) (x${qty})`)
```

Reply:
Also moved the descriptions array out of the startCase. Some elements in the description array are `12"` as in 12in vinyl and startCase was removing the " symbol. This refactor helped solve that problem.